docs/pages/choose-an-edition/teleport-team.mdx

```diff
@@ -39,194 +39,232 @@ web & tsh--Client traffic-->proxy
+   your container's shell with **Ctrl-C**, then run `teleport start` to restart the process.
+   
+   </Notice>
+    
+1. Execute some commands in your terminal. 
+
+1. Enter the `exit` command to end the remote interactive session and close the browser tab.
+
+1. In the last step of **Enroll New Resource**, click **Finish**.
+
+1. Click **Browse existing resources** to see your new server listed in the Teleport Web UI.
+
+## Step 4/5. Play back your session
 
 As Teleport Team proxies SSH connections to registered servers, it records the
 commands that users execute during their sessions so operators can play them
 back later to investigate issues.
 
-In the Teleport Web UI, set the dropdown menu on the upper left of the screen to
-**Management**, then click **Session Recordings**. You will see your session
-from the previous step:
-
-![Session
-recordings](../../img/cloud/getting-started/session-recordings@2x.png)
-
-Click **PLAY**. You will see a full recording of your session.
-
-### Step 4/4. Access your server from the command line
-
-Open a new terminal window and log in to your Teleport Team tenant
-via the command line, filling in the URL of your tenant and the name of your
-Teleport user:
-
-```code
-$ tsh login --proxy=<Var name="tenant-url" />.teleport.sh --user=<Var name="username" />
-```
-
-List the servers your Teleport user can access. You should see the name of the
-container you just registered:
-
-```code
-$ tsh ls
-Node Name    Address    Labels
------------- ---------- ----------------------------------------------------------------------------------------
-000000000000 ⟵ Tunnel   hostname=000000000000,teleport.internal/resource-id=000000000000
-```
-
-Access your server as the `root` user:
-
-```code
-$ tsh ssh root@<Var name="node-name" />
-```
+To playback a session in the Teleport Web UI:
+
+1. Set the dropdown menu on the upper left of the screen to **Management**.
+
+1. Under Activity, click **Session Recordings**.
+   
+   You will see the recording for your interactive session from the previous step listed.
+   For example:
+   
+   ![Session recordings](../../img/cloud/getting-started/session-recordings@2x.png)
+
+1. Click **Play** to see a full recording of your session.
+
+## Step 5/5. Access the server from the command line
+
+To access the server using commands in a terminal:
+
+1. Open a new terminal window.
+
+1. Sign in to your Teleport Team cluster by running the `tsh login` command  with the URL 
+of your cluster and the name of your Teleport user:
+   
+   ```code
+   $ tsh login --proxy=<Var name="cluster-url" />.teleport.sh --user=<Var name="username" />
+   ```
+   
+   When prompted, authenticate using your password, authenticator app, or hardware key.
+   The command displays information about your Teleport Team cluster and account. For example:
+   
+   ```code
+   > Profile URL:      https://cluster-url.teleport.sh:443
```

Comment:
We typically use mytenant.teleport.sh (for cloud) or teleport.example.com (for self hosted). This example is kind of mixing both of them and introducing yet another placeholder value.

Also "cluster-url" is not a great name for a placeholder, because it's not actually a URL.